request-processor-celery/src/application/exceptions/customExceptions.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+
+class URLException(Exception):
```

Comment:
An improvement for future would be a more specific name - i.e. LoadUrlException, UrlFetchException or such